libs/coin-modules/coin-stacks/src/config.ts

```diff
@@ -0,0 +1,26 @@
+import { CurrencyConfig } from "@ledgerhq/coin-framework/config";
+
+export type StacksCoinConfig = () => CurrencyConfig & {
+  config_currency_stacks: {
+    type: "object";
+    default: {
+      status: {
+        type: "active";
+      };
+    };
+  };
+};
```

Comment:
This file can be removed since their is nothing new compare to the default  `CurrencyConfig`